tools/download.py

```diff
@@ -0,0 +1,90 @@
+import argparse
```

Comment:
rename the file from download.py to download_dataset.py and put it to tools/misc

Reply:
Thank you for giving me these suggestions, I will modify this script soon.